### DISCUSSION TITLE
Tauron udostępnił bilansowanie w eLiczniku

### DISCUSSION THREAD

**Discussion by Tikosso** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#164] [Ideas]
Witam
Przydałoby się wprowadzić pobieranie zbilansowanej energii w integracji - podobno jest bardziej wiarygodne, gdyż tak jest energia rozliczana na fakturze.
Pozdrawiam
TP

**Comment by PiotrMachowski** [comment]
Na ten moment nie planuję dodawać tej funkcjonalności, tym bardziej, że wg doniesień wartości zgadzają się idealnie z tymi wyliczonymi przez integrację

**Comment by Tikosso** [comment]
Nie wiem, jak to wygląda na netbillingu, ale na netmeteringu mam jednak inne wartości "magazynu" przed zbilansowaniem i po.

**Reply by kocur80** [reply]
To prawda, żeby obliczyć właściwą liczbę kWh w banku należy znać wartość oddanej energii po zbilansowaniu.
